--- pyblock-0.30/Makefile~	2007-12-14 15:51:04.000000000 +0100
+++ pyblock-0.30/Makefile	2007-12-14 15:52:52.197978950 +0100
@@ -5,7 +5,7 @@
 DEBUGLIB := "/usr/lib/debug"
 CC := gcc
 PYVER := $(shell python -c "import sys; print sys.version[0:3]")
-CFLAGS := -g -I/usr/include/python$(PYVER) -Wall -Werror -fPIC
+CFLAGS := $(OPTFLAGS) -I/usr/include/python$(PYVER) -Wall -Werror -fPIC
 LDFLAGS := -shared
 
 VERSION = 0.30
