--- pyblock-0.15/dm.c.orig	2006-02-08 20:43:08.000000000 +0100
+++ pyblock-0.15/dm.c	2006-05-20 10:09:37.897519250 +0200
@@ -1466,7 +1466,7 @@
 static PyObject *
 pydm_maps(PyObject *self)
 {
-	struct dm_task *task;
+	struct dm_task *task = NULL;
 	struct dm_names *names;
 	int n;
 	unsigned int next = 0;
@@ -1535,7 +1535,7 @@
 static PyObject *
 pydm_targets(PyObject *self)
 {
-	struct dm_task *task;
+	struct dm_task *task = NULL;
 	struct dm_versions *version, *last_version;
 	int n;
 	PyObject *list = NULL, *ret = NULL;
